parsely_raw_data/schema.py

```diff
@@ -141,7 +141,7 @@
     {"key": "version", "ex": 1, "type": int},
     {"key": "visitor", "ex": True, "type": bool},
     {"key": "visitor_ip", "ex": "108.225.131.20", "type": str, "size": 256, "available_with_field": "visitor"},
-    {"key": "visitor_network_id", "ex": "ac94fe31-a307-4020-9a23-fa4798217b02", "type": str, "size": 128, "available_with_field": "visitor"},
+    {"key": "visitor_network_id", "ex": None, "type": str, "size": 128, "available_with_field": "visitor"},
```

Comment:
`None` is correct - that's how it will appear if it's null or set to None. However this should be removed: `"available_with_field": "visitor"`

Reply:
Isn't that still true, considering that this field won't appear at all if Visitor isn't there?  If visitor is there, this appears and is always `None`.

Reply:
All fields appear for every event after the 2.3.0 release padded with Nones. This gives customers a consistent schema to expect

Reply:
Oh, right! Should I just remove the `available_with_field` on all the fields?

Reply:
I think the `available_with_field` is useful for users to know when a field should be Null or not. In this case, visitor_network_id will always be null and shouldn't be available with any field correct?

Reply:
Sounds good to me! I'll change this accordingly.